pkg/models/cloud.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package models
+
+type CloudConfig struct {
+	InstanceIDs   []string
+	PublicIPs     []string
+	Region        string
+	KeyPair       string
+	SecurityGroup string
+	CertFilePath  string
+	ImageID       string
+}
+
+type InstanceConfig struct {
+	Prefix            string
+	CertName          string
+	SecurityGroupName string
+	NumNodes          int
+	InstanceType      string
+}
+
+type CloudConfigMap map[string]CloudConfig
+
+type InstanceConfigMap map[string]InstanceConfig
+
+// GetRegions returns a slice of strings representing the regions of the CloudConfigMap.
+func (ccm *CloudConfigMap) GetRegions() []string {
```

Comment:
maybe this is not needed, and just use maps.Keys whenever needed

Reply:
but replace implementation with maps.Keys